docs/pages/routing/layouts.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Layout Routes
+description: Learn how to define shared UI elements like tab bars and headers.
+hideFromSearch: true
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+import { GithubIcon, BookOpen02Icon, Rocket02Icon } from '@expo/styleguide-icons';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
+
+By default, routes fill the entire screen. Moving between them is a full-page transition with no animation. In native apps, users expect shared elements like headers and tab bars to persist between pages. These can be created using **layout routes**.
+
+## Create a layout route
+
+To create a layout route for a directory, create a file named **\_layout.js** in the directory, and export a React component as `default`.
+
+```js app/home/_layout.js
+import { Slot } from 'expo-router';
+export default function HomeLayout() {
+  return <Slot />;
+}
+```
+
+From the above example, **Slot** will render the current child route, think of this like the `children` prop in React. This component can be wrapped with other components to create a layout.
+
+```js app/home/_layout.js
+import { Slot } from 'expo-router';
+export default function HomeLayout() {
+  return (
+    <>
+      <Header />
+      <Slot />
+      <Footer />
+    </>
+  );
+}
+```
+
+Expo Router supports adding a single layout route for a given directory. If you want to use multiple layout routes, add multiple directories:
+
+<FileTree files={['app/_layout.js', 'app/home/_layout.js', 'app/home/index.js']} />
+
+```js app/_layout.js
+import { Tabs } from 'expo-router';
+
+export default function Layout() {
+  return <Tabs />;
+}
+```
+
+```js app/home/_layout.js
+import { Stack } from 'expo-router';
+
+export default function Layout() {
+  return <Stack />;
+}
+```
+
+If you want multiple layout routes without modifying the URL, you can use [groups](#groups).
+
+## Groups
```

Comment:
This section overlaps in terms of duplicated info from this section: https://github.com/expo/expo/pull/23153/files#diff-ca58a0cc69465f31947e52981ff3361c0f38e0d959beb4ed78ab00ea886d28b6R11

Maybe, we can have that info in one place and then link to the info from the other place.